base/applications/sndrec32/lang/ro-RO.rc

```diff
@@ -16,45 +16,45 @@ END
 
 IDR_MENU1 MENU
 BEGIN
-    POPUP "&Fișier"
+    POPUP "&Fişier"
     BEGIN
         MENUITEM "&Nou", ID_FILE_NEW
         MENUITEM "&Deschidere…", ID_FILE_OPEN
-        MENUITEM "S&alvează", ID_FILE_SAVE, GRAYED
-        MENUITEM "Sal&vează ca…", ID_FILE_SAVEAS, GRAYED
-        MENUITEM "Re&constituire…", ID_FILE_RESTORE, GRAYED
-        MENUITEM "Pr&oprietăți…", ID_FILE_PROPERTIES
+        MENUITEM "&Salvare", ID_FILE_SAVE, GRAYED
+        MENUITEM "S&alvare ca…", ID_FILE_SAVEAS, GRAYED
+        MENUITEM "&Revenire la ultima versiune salvată…", ID_FILE_RESTORE, GRAYED
```

Comment:
Doesn't the menu look horrible to the users eye if a string that long is used?

Reply:
I don't have any problem with that. It's better than before.

![2024-04-13 11_35_30-Resource Hacker - sndrec32 exe mui](https://github.com/reactos/reactos/assets/113553683/a74a1837-c360-40e0-ad02-d54eb91e5709)